go.mod

```diff
@@ -31,108 +29,110 @@ require (
 	golang.org/x/crypto v0.15.0
 	golang.org/x/text v0.14.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.27.3
-	k8s.io/apiextensions-apiserver v0.27.3
-	k8s.io/apimachinery v0.27.3
-	k8s.io/cli-runtime v0.27.3
-	k8s.io/client-go v0.27.3
-	k8s.io/component-base v0.27.3
+	k8s.io/api v0.28.4
+	k8s.io/apiextensions-apiserver v0.28.4
+	k8s.io/apimachinery v0.28.4
+	k8s.io/cli-runtime v0.28.4
+	k8s.io/client-go v0.28.4
+	k8s.io/component-base v0.28.4
 	k8s.io/klog/v2 v2.100.1
-	k8s.io/utils v0.0.0-20230220204549-a5ecb0141aa5
+	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
 	sigs.k8s.io/aws-iam-authenticator v0.6.12
-	sigs.k8s.io/cluster-api v1.5.3
-	sigs.k8s.io/cluster-api/test v1.5.3
-	sigs.k8s.io/controller-runtime v0.15.1
-	sigs.k8s.io/kustomize/api v0.13.4
+	sigs.k8s.io/cluster-api v1.6.0-beta.1
```

Comment:
@hzxuzhonghu Are you able to pick this up again and update to the GA v1.6.0 release?

Reply:
Sorry, i think i met some problems  https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/4569#issuecomment-1821941029

Does 1.6.0 fix it